src/tokenizer.rs

```diff
@@ -644,307 +695,323 @@ fn get_plain_reserved_one_token(input: &str) -> IResult<&str, Token<'_>> {
             terminated(tag("CONCURRENT"), end_of_word),
             terminated(tag("CONSTRAINT"), end_of_word),
             terminated(tag("CONTAINS"), end_of_word),
-            terminated(tag("CONVERT"), end_of_word),
-            terminated(tag("CREATE"), end_of_word),
-            terminated(tag("CROSS"), end_of_word),
             alt((
```

Comment:
Can this nested `alt` be collapsed? iirc the reason we had nested `alt`s is because there was a limit to how many branches could be in a single `alt`. (I suspect some of the very largest branches, such as "s", may not be able to be collapsed, but this still looks like a very nice improvement)

Reply:
There is a limit of 20 or 21 items afaik, the only thing that comes to my mind is to match on next letters in these branches, for example here to check if it,s o or else the rest. I did not do it to leave the order of the keywords as is